internal/meta/branch.go

```diff
@@ -152,23 +143,27 @@ func Trunk(tx ReadTx, name string) (string, bool) {
 	return "", false
 }
 
-func RebuildChildren(tx WriteTx) {
+// Children returns all the immediate children of the given branch.
+func Children(tx ReadTx, name string) []Branch {
 	branches := tx.AllBranches()
```

Comment:
This creates a copy of the map so is probably kind of expensive, but also I don't think it matters at all given that there's essentially zero chance that any reasonable user will ever surpass 1000 branches.